tests/snuba/api/endpoints/test_organization_events_v2.py

```diff
@@ -2474,3 +2475,184 @@ def test_limit_number_of_fields(self):
+        )
+
+    def test_measurements_aggregations(self):
+        self.store_event(self.transaction_data, self.project.id)
+
+        # should try all the potential aggregates
+        query = {
+            "field": [
+                "percentile(measurements.fcp, 0.5)",
+                "count_unique(measurements.fcp)",
+                "min(measurements.fcp)",
+                "max(measurements.fcp)",
+                "avg(measurements.fcp)",
+                "sum(measurements.fcp)",
+            ],
+        }
+        response = self.do_request(query)
+
+        assert response.status_code == 200, response.content
+        assert len(response.data["data"]) == 1
+        assert (
+            response.data["data"][0]["percentile_measurements_fcp_0_5"]
+            == self.transaction_data["measurements"]["fcp"]["value"]
+        )
+        assert response.data["data"][0]["count_unique_measurements_fcp"] == 1
+        assert (
+            response.data["data"][0]["min_measurements_fcp"]
+            == self.transaction_data["measurements"]["fcp"]["value"]
+        )
+        assert (
+            response.data["data"][0]["max_measurements_fcp"]
+            == self.transaction_data["measurements"]["fcp"]["value"]
+        )
+        assert (
+            response.data["data"][0]["avg_measurements_fcp"]
+            == self.transaction_data["measurements"]["fcp"]["value"]
+        )
+        assert (
+            response.data["data"][0]["sum_measurements_fcp"]
+            == self.transaction_data["measurements"]["fcp"]["value"]
+        )
+
+    def get_measurement_condition_response(self, query_str, field):
+        query = {
+            "field": ["transaction", "count()"] + (field if field else []),
+            "query": query_str,
+        }
+        response = self.do_request(query)
+        assert response.status_code == 200, response.content
+        return response
+
+    def assert_measurement_condition_without_results(self, query_str, field=None):
+        response = self.get_measurement_condition_response(query_str, field)
+        assert len(response.data["data"]) == 0
+
+    def assert_measurement_condition_with_results(self, query_str, field=None):
+        response = self.get_measurement_condition_response(query_str, field)
+        assert len(response.data["data"]) == 1
+        assert response.data["data"][0]["transaction"] == self.transaction_data["metadata"]["title"]
+        assert response.data["data"][0]["count"] == 1
+
+    def test_measurements_conditions(self):
+        self.store_event(self.transaction_data, self.project.id)
+
+        # the conditions only work with integer values at the moment
+        fcp = int(self.transaction_data["measurements"]["fcp"]["value"])
+
+        # ============================== Equality ==============================
+        # The parser only works with integer values in these conditions right now
+        # so we cannot filter for equality on floating point values.
```

Comment:
Are there tests missing here?

Reply:
The current parser only supports integers when parsing numeric filters. And since these are floating point values, the parser will error and the test will fail. So I intended to leave the equality comparison out for the time being. Should I write the tests any ways and have them commented out? Or is there an existing pattern for this? I saw that `@pytest.mark.xfail` is used in some places but I wasn't sure if it was a good idea to break apart all of these tests.

Reply:
Not supporting decimal values seems like a problem we'll need to address. I think customers will want to be able to search on more precise values.

Reply:
I'll look into adding it into this PR 👍

Reply:
Addressed in 68d7c84